aclocal.m4

```diff
@@ -100,7 +100,7 @@ dnl Check to see whether a particular set of modules exists. Similar to
 dnl PKG_CHECK_MODULES(), but does not set variables or print errors.
 dnl
 dnl Please remember that m4 expands AC_REQUIRE([PKG_PROG_PKG_CONFIG])
-dnl only at the first occurence in configure.ac, so if the first place
+dnl only at the first occurrence in configure.ac, so if the first place
```

Comment:
Shouldn't `configure` be regenerated now?

Reply:
I don't know and could not do so.  Perhaps this should be reverted and either done separately, for 3.8 only, or left alone.

Reply:
All 3 changes made.  Anyone who wants to fix 'occurence' in aclocal.m4 for 3.8 and regenerate `configure` or whatever should do so separately.